src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.html

```diff
@@ -397,7 +397,7 @@ <h3>Findings</h3>
           other offence</span>
       </div>
       <div *ngIf="!jjDisputedCount"><span class="section-grid-header">&nbsp;<br /></span></div>
-      <div style="width:100%; margin-left: 20px">
+      <div style="width:90%; margin-left: 20px">

```

Comment:
Is there a better way than this? This doens't seem like it will scale well for displays of different sizes (desktop, tablet, phone). I always thought < 100% is considered bad practice, but I don't know. I usually try to set to 100% and use margin or padding instead of parent components.

Reply:
`vw`

Reply:
I guess as long as it looks okay in other resolutions. It just a little strange to have a variable width from the dividing line between counts. I just like things to line up.

Reply:
Once the standard is set we can do that altercations globally how the items need to be aligned rather than doing at element level